filament/src/materials/blit.mat

```diff
@@ -0,0 +1,30 @@
+material {
+    name : blit,
+    parameters : [
+        {
+            type : sampler2d,
+            name : color,
+            precision: high
+        }
+    ],
+    variables : [
+        vertex
+    ],
+    depthWrite : false,
+    depthCulling : false,
+    domain: postprocess
+}
+
+vertex {
+    void postProcessVertex(inout PostProcessVertexInputs postProcess) {
+        postProcess.vertex.xy = postProcess.normalizedUV;
+    }
+}
+
+fragment {
+    void postProcess(inout PostProcessInputs postProcess) {
+        vec2 uv = variable_vertex.xy; // interpolated to pixel center
```

Comment:
It should be `HIGHP vec2 uv = …` since it's UVs.

Reply:
yes!